app/models/package_manager/maven.rb

```diff
@@ -106,7 +112,7 @@ def self.mapping_from_pom_xml(version_xml, depth = 0)
           xml.locate('scm/url/?[0]').first,
           xml.locate('url/?[0]').first
         ),
-        licenses: licenses(xml).join(","),
+        licenses: licenses(version_xml).join(","),
```

Comment:
Isn't `xml` the "canonical" xml in this method?

Reply:
Nope - we call `version_xml.project` which doesn't include the top-level comment (line 84)